drupal-check

```diff
@@ -21,7 +21,7 @@ else {
     exit(1);
 }
 
-$xdebug = new XdebugHandler('cua', '--ansi');
+$xdebug = new XdebugHandler('cua');
```

Comment:
One idea is to keep this as is, and then require `composer/xdebug-handler:^1.3 || ^2.0.1`. I think that would make `mglaman/drupal-check` play better with dependencies whether installed on D8 or D9.

Reply:
I'm fine with just v2 and not having to support both

Reply:
Ah, we did need both, whoops.